src/js/labels/type.ts

```diff
@@ -0,0 +1,22 @@
+/**
+ * LabelNested
+ * @prop color - the color of the label
+ * @prop description - the description of the label
+ * @prop id - the unique identifier of the label
+ * @prop name - the name of the label
+ */
+export type LabelNested = {
+    color: string;
+    description: string;
+    id: number;
+    name: string;
+};
```

Comment:
```suggestion

/*Smaller label object for representing labels in a list */
export type LabelNested = {
    /* A hex encoded color*/
    color: string;
    /*A detailed description*/
    description: string;
    <ect>
    id: number;
    <ect>
    name: string;
};
```
Few general tweaks:
  1. Descriptions of each field in a type should be right above the field
  2. The overall description should just be a one liner describing the overall object
  3. Update the Type descriptions to include more helpful info. Many of the current docstrings don't add much information that couldn't be grabbed at a glance.

Please rework the descriptions and fill out remaining fields not changed here